scripts/bert/index.rst

```diff
@@ -74,4 +77,15 @@ Should produce an output like this. Explain that the F1 score on the dev dataset
 
 .. code-block:: console
 
-    {'exact_match': 81.21097445600756, 'f1': 88.4551346176558}
\ No newline at end of file
+    {'exact_match': 81.21097445600756, 'f1': 88.4551346176558}
+
+SQuAD 2.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are pre-training on the SQuAD2.0 dataset, you need to specify the parameter **version_2** and specify the parameter **null_score_diff_threshold** (Typical values are between -1.0 and -5.0).
+
+Use the following command to fine tune the BERT model of the SQuAD2.0 dataset and generate predictions.json, nbest_predictions.json, and null_odds.json.
+
+.. code-block:: console
+
+    $ python finetune_squad.py --optimizer adam --batch_size 12 --lr 3e-5 --epochs 2 --null_score_diff_threshold -2.0 --gpu --version_2
```

Comment:
Forgot to append the results for SQuAD 2.0?

Reply:
The original repo did not provide the results of squad2.0, do we need to provide it?

Reply:
You can check whether the length of data is reasonable by outputting ` batch_sampler.stats() `. If have a lot of long samples, you may still need to clip them.

Reply:
I think it would be great if we have the results for 2.0. Is there any blocker?

Reply:
@cgraywang The reported squad 2.0 test f1 is using BERT LARGE and uses lots of memory. Also the official repo didn't report the dev f1 on squad 2.0. Maybe we can set this as a stretch goal for this PR.

Reply:
I think we can run the experiments using BERT-base. It would be good to give the users the expectations if they want to use squad 2.0 and want to know whether they are generating reasonable results. Another option could be removing the squad 2.0 option in this script.

Reply:
Ok, I will give the results of the Large model and the training log later.